README.md

```diff
@@ -39,7 +39,7 @@ The above packages follow [Semantic Versioning](https://semver.org). Consult the
 
 We welcome feedback and contributions!
 
-The fastest way to ask questions is to [join the discussion on Teams](https://teams.microsoft.com/l/team/19%3awo8vmMKMsHfltKXxc0bczZo-X4JlQSV5VxpaRJdh13k1%40thread.tacv2/conversations?groupId=9ee065d7-3898-4245-82f6-76e86084b8b1&tenantId=87ba1f9a-44cd-43a6-b008-6fdb45a5204e) (accessible to NI employees only).
+The fastest way to ask questions is to [join the discussion on Teams](https://teams.microsoft.com/l/team/19%3awo8vmMKMsHfltKXxc0bczZo-X4JlQSV5VxpaRJdh13k1%40thread.tacv2/conversations?groupId=9ee065d7-3898-4245-82f6-76e86084b8b1&tenantId=87ba1f9a-44cd-43a6-b008-6fdb45a5204e) (accessible to NI employees only). You can also [open a discussion](https://github.com/ni/nimble/discussions) on GitHub.
 
 ## Requesting New Components and Features
```

Comment:
Our venues for discussing design system related issues feel pretty fractured to me:

- Microsoft teams
  - Design System Team Dev Squad and General
  - ASW SystemLink - UI Working Group
- NI Stack Overflow
- GitHub Issues
- AzDo Issues

I don't think we are quite ready for a general outside of NI community and I don't really want to fork the discussions. 

Longer-term I can see the discussions tab making sense but if that's the direction we want to go then I'd like to advocate / discuss redirecting some of those other venues to the discussions tab as well. If it is just a location for non-NI people to ask questions that makes it less valuable, making it the goto for NI discussions would make it a more useful repository of information. I.e. you see microsoft employees popping up and asking questions in the public fast discord frequently.

I'd advocate we enable the discussions tab when we feel like they will add known value. Like, for example, directing folks making GitHub issues which just ask question to use discussion forums instead. Either that or we use it as a way to consolidate some conversation venues and make it the goto source of questions / truth.

But none of that is something I'm hard set on. I just think turning it on at this point without a plan to bring more discussion there is unlikely to make it successful.

Reply:
While I don't believe we will get much traction on the Discussions area for now, it is good to have some way for the public (non-NI employees) to provide questions and input without necessarily having to create an issue. 

If it begins to get muddy or we start seeing things pop-up from internal folks and that becomes unwieldy, we can create a more formal communication on where to submit what type of request.

Reply:
We decided in our team meeting to:
1. disable discussions
2. create an issue template for discussions / asking for help
3. update this to link to that issue template

We'll wait and see what kind of activity we get before deciding whether to move to GitHub Dicussions, Discord, or some other platform.

I'll make these changes.

Reply:
@rajsite I updated the PR to reflect this decision so it's ready for your next round of review.